src/control/README.md

```diff
@@ -14,10 +14,11 @@ The [shell](dmg/daos_shell) is an example client application which can connect t
 
 ## Documentation
 
-- [Management API](https://godoc.org/github.com/daos-stack/daos/src/control/client)
-- [Management internals](https://godoc.org/github.com/daos-stack/daos/src/control/server)
-- [Agent API](https://godoc.org/github.com/daos-stack/daos/src/control/client/agent)
-- [Agent internals](https://godoc.org/github.com/daos-stack/daos/src/control/security)
+- [Management API](/src/control/client)
```

Comment:
you have changed the URLs, Will this still show the godoc representation of the Apis?

Reply:
Good catch - I thought these were just going to the README.md files. I'll revert.